docs/etcd_clusters.md

```diff
@@ -0,0 +1,60 @@
+# Manage etcd clusters
+
+Mayu contains an etcd discovery to setup and manage your etcd clusters. During startup you can configure if you like to use an external etcd discovery or Mayu itself.
```

Comment:
I am confused now. Does Mayu always provide an etcd-discovery compatible API? And you can optionally configure the cluster mgmt to use that API or the official one?

I was under the impression we always proxy the etcd-discovery stuff through mayu, but forward it to an external service (`dicovery.etcd.io` or your own etcd).